pkgs/tools/graphics/exifprobe/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "exifprobe";
+  version = "unstable-2018-06-19";
+
+  src = fetchFromGitHub {
+    owner = "hfiguiere";
+    repo = "exifprobe";
+    rev = "ce1ea2bc3dbbe8092b26f41cd89831cafe633d69";
+    sha256 = "1c1fhc0v1m452lgnfcijnvrc0by06qfbhn3zkliqi60kv8l2isbp";
+  };
+
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  postInstall = ''
+    mv $out/usr/bin $out/bin
```

Comment:
Is install a good alternative here?

Reply:
How would that look?

Reply:
``install -Dm755 $out/usr/bin/* -t $out/bin``

Reply:
I seem to get the following error when I use `install`

```
./mkcomptime > comptime.c
clang -o exifprobe -O2 -DCOLOR -O -std=c99   main.o readfile.o print.o tagnames.o process.o misc.o interpret.o print_maker.o maker_generic.o maker_tagnames.o options.o ciff.o jp2000.o mrw.o veclib.o raf.o x3f.o maker_nikon.o maker_epson.o maker_olympus.o maker_fujifilm.o maker_casio.o maker_minolta.o maker_sanyo.o maker_canon.o maker_leica.o maker_agfa.o maker_asahi.o maker_panasonic.o maker_kyocera.o maker_toshiba.o maker_sigma.o maker_sony.o maker_konica.o maker_hp.o maker_kodak.o maker_ricoh.o maker_traveler.o comptime.c -lm
ld: warning: passed two min versions (10.12.0, 10.12) for platform macOS. Using 10.12.
installing
install flags: SHELL=/nix/store/8i9kyqfxhk2s9dzx32x0sgdk7kyx9wjc-bash-4.4-p23/bin/bash DESTDIR=\$\(out\) install
install -d /nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/bin
install -d /nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/share/man/man1
install -c -m 755 exifprobe /nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/bin
install -c -m 755 exifgrep /nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/bin
install -c -m 644  exifprobe.1 /nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/share/man/man1
install -c -m 644  exifgrep.1 /nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/share/man/man1
install: omitting directory '/nix/store/aq947n6p7irs8cj94dmfnia80fsf88hk-exifprobe-unstable-2018-06-19/usr/share/man'
builder for '/nix/store/7adsqa2rgmif6srgmvsrxc8shhdh3gqb-exifprobe-unstable-2018-06-19.drv' failed with exit code 1
error: build of '/nix/store/7adsqa2rgmif6srgmvsrxc8shhdh3gqb-exifprobe-unstable-2018-06-19.drv' failed
```